src/cmd/manage_nsfs.js

```diff
@@ -218,9 +230,18 @@ async function fetch_bucket_data(argv, config_root, from_file) {
             new_name: argv.new_name
         };
     }
+
+    if (argv.bucket_policy !== undefined) {
```

Comment:
@romayalon are we passing policy in manage_nsfs arguments? I couldnt find it in help

Reply:
I forgot to add it to the help, will add it :) thanks!